Rakefile

```diff
@@ -903,10 +903,60 @@ task :compile do
   end
 end
 
+# Used for kokoro
+# ====================================================================================================================================
+namespace :kokoro do
+  ruby_versions = ['2.3', '2.4', '2.5']
+  
+  desc "Generate presubmit configs for kokoro"
+  task :builds do
+    generate_kokoro_configs ruby_versions
+  end
+end
+
+def generate_kokoro_configs ruby_versions
+  gems.each do |gem|
+    #  generate the presubmi configs
+    File.open("./.kokoro/presubmit/#{gem}.cfg", 'w') do |f| 
+      f.write(kokoro_config(gem))
+    end
+
+    # generate the continuous configs
+    ruby_versions.each do |ruby_version|
+      File.open("./.kokoro/continuous/#{gem}-ruby-#{ruby_version}.cfg", 'w') do |f| 
+        f.write(kokoro_config(gem, ruby_version))
+      end
+    end
+
+  end
+end
+
+def kokoro_config gem, ruby_version = nil
+  lines = []
```

Comment:
Thoughts on using an ERB file instead of putting this inline?

Reply:
I'm for it.